util/prep-bazel-airgapped-build.sh

```diff
@@ -167,7 +167,7 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
   readonly LATEST_BISTREAM_HASH_FILE="${SYSTEM_BITSTREAM_CACHE}/latest.txt"
   # The revision named in latest.txt is not necessarily on disk. Induce the
   # cache backend to fetch the latest bitstreams.
-  BITSTREAM=latest ${BAZELISK} fetch @bitstreams//...
+  BITSTREAM="latest" ${BAZELISK} fetch @bitstreams//...
```

Comment:
I don't think using the "latest" tag is wise in general. What's the context for using this instead of "HEAD"? Is the airgapped variant not still a git repo?

Reply:
There is no context. It was simply what already existed on both the [earlgrey_1.0.0](https://github.com/lowRISC/opentitan/blob/676fe222252f7a9fc4e0f4b2a4c14c53dfff1b96/util/prep-bazel-airgapped-build.sh#L170) branch and [master](https://github.com/lowRISC/opentitan/blob/fb326fc736e955f3a6f2d8db44de03c430b9b9b1/util/prep-bazel-airgapped-build.sh#L155) branch. Happy to update it to HEAD in a future change. This PR simply switches the branch folder the cache setup script uses to fix the airgapped DV environment

Reply:
That's fine, but if it had used HEAD, the original motivation for this PR wouldn't have existed, hehe. The airgapped build would've retrieved a bitstream that was part of the earlgrey_1.0.0 branch, and there would be no issues.

There's not a problem with this PR or anything; I'm just showing you some context and that the tech debt came from the choice of ever creating this "latest" pointer, refreshing your memory on stuff you and I talked about years ago. :smile: 

I'm sorry I didn't fix it already--I've paid very little attention to this CI job and didn't know it was still using that label. We changed the default in the script itself awhile back: https://github.com/lowRISC/opentitan/pull/14348#discussion_r951722702